src/main/java/io/debezium/connector/jdbc/naming/TableNamingStrategy.java

```diff
@@ -15,6 +15,7 @@
  * @author Chris Cranford
  */
 public interface TableNamingStrategy {
+    String IGNORE_SINK_RECORD_FOR_TABLE = "__IGNORE_SINK_RECORD_FOR_TABLE";
```

Comment:
I don't think we need a magical constant. IMHO the better soltuion would be to extend `TableNamingStrategy` contract (and document in JavaDoc) that if the returned value is `null`, then the record should be dropped.

Reply:
@jpechane thanks! Change to use `null` instead of magic constant.